src/UNIVesting.sol

```diff
@@ -0,0 +1,121 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.29;
+
+import {
+  BokkyPooBahsDateTimeLibrary as DateTime
+} from "BokkyPooBahsDateTimeLibrary/contracts/BokkyPooBahsDateTimeLibrary.sol";
+import {Owned} from "solmate/src/auth/Owned.sol";
+import {ERC20} from "solmate/src/tokens/ERC20.sol";
+import {SafeTransferLib} from "solmate/src/utils/SafeTransferLib.sol";
+import {IUNIVesting} from "./interfaces/IUNIVesting.sol";
+
+/// @title UNIVesting
+/// @notice A vesting contract that releases UNI tokens quarterly to a designated recipient
+/// @dev The contract starts vesting on January 1, 2026 and allows withdrawals every calendar
```

Comment:
This is now not true - The vesting starts 3 months prior to Jan 1 since you initialize lastUnlockTimestamp to 3 months before. Meaning you can call withdraw ON Jan 1 and successfully withdraw the first quarter allotment that has been vesting over the last 3 months

Imo vesting should begin Jan 1, so first unlock should happen 3 months later. But would update this comment bc thats not the behavior rn.